Description: Upstream changes introduced in version 1.4.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 openvswitch (1.4.0-2) unstable; urgency=low
 .
   * Use explicit DKMS variable for kernel source directory
     - 0001-debian-Fix-dependencies-for-openvswitch-datapath-dkm.patch
     - 0002-debian-Look-in-lib-modules-instead-of-usr-src-for-DK.patch
     - 0001-debian-Use-provided-kernel-source-dir.patch
     - 0001-debian-Do-not-run-make-if-configure-fails-during-DKM.patch
     - (closes: #659685)
   * Don't install Python modules for obsolete Python versions
     - 0001-debian-Don-t-install-Python-modules-for-obsolete-Pyt.patch
 .
 The person named in the Author field signed this changelog entry.
Author: Simon Horman <horms@debian.org>
Bug-Debian: http://bugs.debian.org/659685

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- openvswitch-1.4.0.orig/AUTHORS
+++ openvswitch-1.4.0/AUTHORS
@@ -89,6 +89,7 @@ Joan Cirer              joan@ev0.net
 John Galgay             john@galgay.net
 Koichi Yagishita        yagishita.koichi@jrc.co.jp
 Krishna Miriyala        krishna@nicira.com
+Luca Falavigna          dktrkranz@debian.org
 Luiz Henrique Ozaki     luiz.ozaki@gmail.com
 Michael A. Collins      mike.a.collins@ark-net.org
 Michael Hu              mhu@nicira.com
--- openvswitch-1.4.0.orig/debian/openvswitch-test.install
+++ openvswitch-1.4.0/debian/openvswitch-test.install
@@ -1,2 +1,2 @@
-usr/share/openvswitch/python/ovstest usr/lib/python2.6/dist-packages/
+usr/share/openvswitch/python/ovstest usr/lib/python2.4/site-packages/
 usr/bin/ovs-test
--- openvswitch-1.4.0.orig/debian/control
+++ openvswitch-1.4.0/debian/control
@@ -25,7 +25,7 @@ Description: Open vSwitch datapath modul
 
 Package: openvswitch-datapath-dkms
 Architecture: all
-Depends: dkms (>= 1.95), make, libc6-dev, ${misc:Depends}
+Depends: dkms (>= 1.95), make, ${misc:Depends}
 Description: Open vSwitch datapath module source - DKMS version
  This package provides the Open vSwitch datapath module source code
  that is needed by openvswitch-switch.  DKMS can built the kernel
--- openvswitch-1.4.0.orig/debian/dkms.conf.in
+++ openvswitch-1.4.0/debian/dkms.conf.in
@@ -1,6 +1,6 @@
 PACKAGE_NAME="openvswitch"
 PACKAGE_VERSION="__VERSION__"
-MAKE="./configure --with-linux='${kernel_source_dir}' && make -C datapath/linux"
+MAKE="./configure --with-linux=/usr/src/linux-headers-`uname -r` ; make -C datapath/linux"
 BUILT_MODULE_NAME[0]=openvswitch_mod
 BUILT_MODULE_NAME[1]=brcompat_mod
 BUILT_MODULE_LOCATION[0]=datapath/linux/
--- openvswitch-1.4.0.orig/debian/python-openvswitch.install
+++ openvswitch-1.4.0/debian/python-openvswitch.install
@@ -1 +1 @@
-usr/share/openvswitch/python/ovs usr/lib/python2.6/dist-packages/
+usr/share/openvswitch/python/ovs usr/lib/python2.4/site-packages/
